source/release-notes.txt

```diff
@@ -10,6 +10,46 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.41.0
+----------------
+
+*Upcoming* 
```

Comment:
Just double checking, are we going to update this when we merge? We released 1.41.0 on December 18th

Reply:
Merge pushes the changes into the master branch.  Usually someone on the Server Docs goes through and publishes the updates a couple times a day. 

Fixed